MapboxCoreNavigation/NavigationRouteOptions.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+import MapboxDirections
+
+/**
+ A `NavigationRouteOptions` object is a structure that specifies the criteria for results returned by the Mapbox Directions API.
+ 
+ `NavigationRouteOptions` is a subclass of `RouteOptions` that has been optimized for navigation. Pass an instance of this class into the `Directions.calculate(_:completionHandler:)` method.
+ */
+@objc(MBNavigationRouteOptions)
+open class NavigationRouteOptions: RouteOptions {
```

Comment:
I had been thinking of calling this class NavigationOptions, but NavigationRouteOptions makes more sense. 👍